src/getTime.js

```diff
@@ -19,6 +19,9 @@
  */
 function getTime(str) {
   // write code here
+  const timeRE = /\b(2[0-3]|[01]\d):[0-5]\d\b/;
+  let time = timeRE.exec(str);
```

Comment:
suggestion: timeMatches. it's an array, so at least times or timesArr.

Reply:
Thank's a lot!